### ISSUE TITLE
triple click merge with the next cellule

### ISSUE THREAD

**Issue by Goaman** [odoo-dev/jabberwock#129]
Is it the proper behavior that we want?
https://youtu.be/b2hx6G9Vf9I

**Comment by dmo-odoo** [comment]
I don't think we explicitely bound anything on a "triple click" so I guess it's up to the normalizer in this case. Is the triggered setSelection correct ? Besides, you can upload videos to Odoo's drive if you like.

**Comment by Goaman** [comment]
It is indeed the Normalizer and it's doing what the contenteditable is doing but the result seems odd to me. Therefore I think we should think about that case in the future. Thus this issue.
